packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -601,12 +613,18 @@ export const iconMetadata: {
     IconTriangleTwoLinesHorizontal: {
         tags: ['collapse all']
     },
+    IconTwoChatSparkle: {
+        tags: ['nigel-chatbot', 'ai', 'chat', 'help', 'conversation']
+    },
     IconTwoSquaresInBrackets: {
         tags: ['group by']
     },
     IconTwoTrianglesBetweenLines: {
         tags: ['size column to content']
     },
+    IconUndockWindow: {
```

Comment:
Font Awesome [appears to use](https://fontawesome.com/search?q=window&o=r&s=solid&ip=classic) `IconWindowRestore` for this concept.